presto-native-execution/CMakeLists.txt

```diff
@@ -114,6 +114,8 @@ if(PRESTO_ENABLE_CUDF)
   set(VELOX_ENABLE_CUDF
       ON
       CACHE BOOL "Enable cuDF support")
+  set(VELOX_ENABLE_PARSE ON)
```

Comment:
@zoltan Can you explain why we need to `set(VELOX_ENABLE_PARSE ON)` `set(VELOX_ENABLE_DUCKDB ON)` ? IThese dependencies should be resolved on the Velox side. We should also not depend on duckdb other than for testing.

Reply:
Nevermind. You need it for `facebook::velox::parse::parseExpr` below.